tutor/src/screens/assignment-review/details.js

```diff
@@ -240,16 +236,25 @@ const TaskingDates = observer(({ tasking, title }) => {
       <dl>
         <div>
           <dt>Open date</dt>
-          <dd>{format(tasking.opensAtMoment)}</dd>
+          <dd>
+            <div>{tasking.opensAtMoment.format(dateFormat)}</div>
+            <div>{tasking.opensAtMoment.format(timeFormat)}</div>
+          </dd>
         </div>
         <div>
           <dt>Due date</dt>
-          <dd>{format(tasking.dueAtMoment)}</dd>
+          <dd>
+            <div>{tasking.dueAtMoment.format(dateFormat)}</div>
+            <div>{tasking.dueAtMoment.format(timeFormat)}</div>
+          </dd>
         </div>
         {!tasking.plan.isEvent && (
           <div>
             <dt>Close date</dt>
-            <dd>{format(tasking.closesAtMoment)}</dd>
+            <dd>
+              <div>{tasking.closesAtMoment.format(dateFormat)}</div>
+              <div>{tasking.closesAtMoment.format(timeFormat)}</div>
+            </dd>
           </div>)}
```

Comment:
I'm probably missing something?  Why are each of the dates listed twice?

Reply:
It breaks the date and time into separate divs so the line break is nicer than wrapping

Reply:
aha! I completely missed the date/time difference there